utils/luafunc_dup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/bash
+
+LST=`tempfile`
+res=0
+trap 'rm -f $LST ; exit $res' EXIT
+
+for arg in $@ ; do
```

Comment:
```suggestion
for arg in "$@ "; do
```
Avoid re-splitting elements.

Reply:
Absolutely !
That said, I think your `'"'` and `' '` were swapped.

Reply:
Thanks for the notice, I dislike the github editor.